### DISCUSSION TITLE
Support flex gap css property?

### DISCUSSION THREAD

**Discussion by lincolnnguyen18** [ReactUnity/core#67] [Ideas]
<img width="746" alt="image" src="https://user-images.githubusercontent.com/61459043/190705392-5e22eeee-5345-4519-b9fc-46beeaffa21b.png">

I wish the gap property for flex containers could be supported! Although I just found out about `:not(:last-child)` which seems pretty cool!

**Comment by gkurt** [comment]
It is already supported in all platforms except WebGL and IOS. This is a recent addition.

**Reply by alexishecf** [reply]
Do you plan to support WebGL in the future?

**Reply by gkurt** [reply]
ReactUnity uses Yoga library for flexbox layout calculations (including flex gaps). This library is a dynamic library. The thing is, Unity engine itself already includes this library (Because it also uses Yoga in UIElements). And the version they use is an old one which does not support gap property. For this reason, it is very hard for me to add this functionality to these platforms. It would be only possible if Unity implemented this property.
